language.go

```diff
@@ -157,6 +157,9 @@ var Exts = map[string]string{
 	"jai":         "JAI",
 	"java":        "Java",
 	"jsp":         "JSP",
+	"vb":          "VBNet",
+	"resx":        "VBNet",
```

Comment:
I think that should use `XML` or `XML Resource` or `.NET Resource` instead of `VBNet`. Because `.resx` files is resource definition for .NET framework.

ref: https://docs.microsoft.com/en-us/dotnet/framework/resources/creating-resource-files-for-desktop-apps